assets/css/main.css

```diff
@@ -0,0 +1,579 @@
+  position: absolute !important;
+  clip: rect(1px, 1px, 1px, 1px);
+  padding:0 !important;
+  border:0 !important;
+  height: 1px !important;
+  width: 1px !important;
+  overflow: hidden;
+}
+
+.img {
+  width: 100%;
+}
+
+.advisor-select {
+  display: none;
+}
+
+textarea.context-of-advice {
+  display: block;
+  margin: 0 auto;
+}
+
+@media (min-width: 800px) {
+  textarea.context-of-advice {
+    width: 50%;
+  }
+}
+
+.message-area {
+  margin: 0 auto;
+  max-width: 600px;
+  padding: 15px;
+  text-align: center;
+}
+
+blockquote {
+  background: var(--darker-background);
+  border-left: 5px solid var(--darkest-background);
+  font-style: italic;
+  margin: 1.5em 0;
+  padding: .5em 10px;
+}
+
+.message-area > p {
+  text-align: left;
+}
+
+textarea {
+  border: 1px solid var(--border-grey);
+  border-radius: 3px;
+  color: var(--text-color);
+  outline: none;
+  padding: 5px;
+  resize: none;
+  width: 100%;
+}
+
+.completeness {
+  display: block;
+  font-size: 16px;
+  font-weight: bold;
+  line-height: 26px;
+  margin-bottom: 50px;
+  text-align: center;
+}
+
+.profile-image.incomplete {
+  filter: grayscale(100%);
+  opacity: .5;
+}
+
+.provider {
+  margin-top: 25px;
+  position: relative;
+  width: 100%;
+}
+
+.provider > img {
+  border-radius: 22px;
+  transform: translateY(5px);
+  width: 22px;
+  height: 22px;
+}
+
+.provider > span {
+  font-size: 16px;
+  font-weight: bold;
+  line-height: 25px;
+  margin-left: 5px;
+}
+
+.download-link {
+  width: 30px;
+}
+
+.small-circle {
+  width: 48px;
+}
+
+.circle {
+  border-radius: 50%;
+}
+
+.profile-size {
+  width: 200px;
+}
+
+.green {
+  background: var(--green);
+  box-shadow: inset 0 -3px 0 var(--darker-green);
+}
+
+.red {
+  background: var(--red);
+  box-shadow: inset 0 -3px 0 var(--darker-red);
+}
+
+.blue {
+  background: var(--primary);
+  box-shadow: inset 0 -3px 0 var(--darker-primary);
+}
+
+.user-image-input {
+  text-overflow: ellipsis;
+}
+
+.notice {
+  background: var(--notice);
+  border-radius: 0 3px 3px 0;
+  bottom: 0;
+  left: 0;
+  margin: 0 auto;
+  padding: 1em 0;
+  position: fixed;
+  text-align: center;
+  width: 100%;
+}
+
+.invisible {
+  display: none;
+}
+
+@keyframes fade-out {
+  from {
+    opacity: 1;
+  }
+
+  to {
+    opacity: 0;
+  }
+}
+
+.fade-out {
+  animation-duration: 2s;
+  animation-fill-mode: both;
+  animation-name: fade-out;
+  animation-timing-function: cubic-bezier(.755, .05, .855, .06);
+}
+
+.flash-notification {
+  text-align: center;
+  height: 40px;
+  width: 200px;
+  margin: 40px auto 0;
+  padding-top: 10px;
+}
+
+.flash-notification:not(:empty) {
+  animation: background-fade 2s linear 0s;
+}
+
+@keyframes background-fade {
+  from { background-color: #ffb3c9}
+  to {background-color: transparent}
+}
+
+.modal {
+  align-items: center;
+  display: none;
+  justify-content: center;
+  overflow: hidden;
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  z-index: 20;
+}
+
+.modal.is-active {
```

Comment:
I'm not quire sure what approach take https://every-layout.dev/rudiments/modular-scale/ when it came to state, but you could consider http://getbem.com/ where you would have something like
``` css
.modal--active {}
```
it avoid you to chain class name and easily see the style that belongs together.